core/client/src/main/java/alluxio/client/block/BlockStoreContext.java

```diff
@@ -188,9 +191,10 @@ public void releaseWorkerClient(BlockWorkerClient blockWorkerClient) {
    * @return the acquired netty channel
    * @throws IOException if it fails to create a new client instance mostly because it fails to
    *         connect to remote worker
+   * @throws InterruptedException if this thread is interrupted
    */
   public static Channel acquireNettyChannel(final InetSocketAddress address,
-      final Callable<Bootstrap> bootstrapBuilder) throws IOException {
+      final Callable<Bootstrap> bootstrapBuilder) throws IOException, InterruptedException {
```

Comment:
that is for a different function. but yeah, it isn't necessary in acquireNettyChannel. InterruptedException is only necessary for heartbeat in which it accepts interrupts to stop hearbeat.